signals/events/base/src/main/java/org/eclipse/ditto/signals/events/base/Event.java

```diff
@@ -77,6 +78,14 @@ default JsonSchemaVersion getImplementedSchemaVersion() {
      */
     Optional<Instant> getTimestamp();
 
+    /**
+     * Returns the event's metadata.
+     *
+     * @return the metadata.
+     * @since 1.3.0
+     */
+    Optional<Metadata> getMetadata();
```

Comment:
This method fits better in ThingEvent. No other event has metadata.

Reply:
Yes, however we also added getMetadata() to Entity so that eg also Policy could have metadata.
However policy also only provides empty metadata currently.
So this is analog to Entity